ee/ucp/interlock/usage/service-clusters.md

```diff
@@ -45,7 +45,7 @@ map[nodetype:loadbalancer region:us-west]
 Next, we will create a configuration object for Interlock that contains multiple extensions with varying service clusters:
 
 ```bash
-$> cat << EOF | docker config create com.docker.ucp.interlock.conf-1 -
+$> cat << EOF | docker config create com.docker.ucp.interlock.service-clusters.conf-1 -
```

Comment:
The UCP version comment should apply to the config name change as well. The `com.docker.ucp.interlock.service-clusters.conf` value is only required for versions prior to 3.0.10 and 3.1.4

Reply:
@jtestard - Hi Jules. PTAL - thank you for the clarification!